Src/WitsmlExplorer.Api/Extensions/HttpRequestExtensions.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+
+using Microsoft.AspNetCore.Http;
+
+using WitsmlExplorer.Api.Configuration;
+namespace WitsmlExplorer.Api.Extensions
+{
+    public static class HttpRequestExtensions
+    {
+
+        /// The format for WitsmlServer Header value is : "b64(creds)@URL"
+        public static ServerCredentials GetWitsmlServerHttpHeader(this HttpRequest httpRequest, string headerName, Func<string, string> delDecrypt)
+        {
+            string[] headerSplitted = httpRequest.Headers[headerName].ToString().Split('@');
```

Comment:
headerSplit? 😄

Reply:
Sure, I need to upgrade my language. I will fix :D